packages/yoroi-extension/app/connector/stores/ConnectorStore.js

```diff
@@ -631,17 +639,10 @@ export default class ConnectorStore extends Store<StoresMap, ActionsMap> {
       amount: txBody.fee().to_str(),
     };
 
-    const ownAddresses = new Set([
-      ...response.utxos.map(utxo => utxo.address),
-      ...response.usedAddresses,
-      ...response.unusedAddresses,
-      response.changeAddress
-    ])
-
     const { amount, total } = await this._calculateAmountAndTotal(
       selectedWallet.publicDeriver,
       inputs,
-      outputs,
+      outputs.map(({ address, value }) => ({ address, value })),
```

Comment:
`_calculateAmountAndTotal` accepts `outputs` with the type below. So I needed to remove the `isForeign` flag so flow will not complain 

```ts
outputs : Array<{ address: string, value: MultiToken}>
```

Reply:
Thank you, @AhmedIbrahim336 ! I have updated the code now a little, cuz we don't need to do the actual mapping (calculations) just to match the flow types. We can just accept an inexact type in the function if we mark it as read-only, see the change here: https://github.com/Emurgo/yoroi-frontend/pull/3091/commits/198900e27f37cd3917c2e0f3965a1a425f6e3b9e#diff-4fdbc4bd4e71aa6ffc075cb2acf855b5870659d9f1d40e25e1ce6b2cf76af58eR857